draft-thomson-quic-tls.md

```diff
@@ -285,6 +282,10 @@ of keys is used for protecting outbound messages, the KEY_PHASE bit in the
 public flags is toggled.  The KEY_PHASE bit on unencrypted messages is 0.
 
 The KEY_PHASE bit on the public flags is the most significant bit (0x80).
+Note: I don't think we want to make the most significant bit. In fact, I think
+we probably want this to be a bit in the second byte of public flags such
+that when we switch to full encryption, we can stop sending the extra byte
+altogether.
```

Comment:
This is going to need to be on every packet.  You don't want to push this into the second octet.  I can see the version negotiation going into an extra octet, and maybe even an extra length bit for the connection ID, and especially the public reset bit but not this one.